### ISSUE TITLE
SC 3.1.5 Reading Level & lower secondary education level

### ISSUE THREAD

**Issue by LaZay** [w3c/wcag#2852]
As it is defined now, the 3.1.5 Reading Level criteria is not an accessibility criteria. It is an audience criteria. A **large audience criteria**. Which is great for large audience publishing. Which is a catastropha for educational publishing (Upper secondary, Higher education), but also scientific, technical, legal, medical, any specialized publishing as a matter of fact. It will **discourage the respect of every other clever AAA criteria**. Because all AAA criteria must be met to reach this WCAG level. Why even try then? :-(

OK to apply this criteria to large audience sites/publishing. It cannot apply to sites/publications that are specifically written for readers with higher education level than the Lower secondary (MCQs, textbooks, etc. contents). Or that are written for experts (specialized sites/books)... These cases should be exceptions to the criteria. Otherwise NO upper education, scientific, technical, legal, medical, any specialized publishing will never offer any AAA service. I am sure nobody wants this.

Some impaired people go to university, other become great experts. They need the highest level of content accessibility to succeed in their studies/career. This includes all AAA criteria. Except for the 3.1.5 one, as it is defined now.

**Comment by alastc** [comment]
> Reading Level criteria is not an accessibility criteria. It is an audience criteria.

All the guidelines are aimed at characteristics of the content, in this case that the complexity of the language is not above a certain level.

However, I certainly take the point about how applicable that is across all content, particularly for  higher level education or specialised materials (e.g. one of our clients publishes physics journals).

It was before my time, but I'd guess that it why it is a AAA rather than AA. There was an attempt in WCAG 2.1 to form a criterion for this at A or AA level, but it is very difficult to draw lines for which content would be exempt or not, it did not succeed. 

In WCAG 3 work that type of guideline is used as one of the examples so that we can establish how best to incorporate it. 

> Some impaired people go to university, other become great experts. They need the highest level of content accessibility to succeed in their studies/career. This includes all AAA criteria. Except for the 3.1.5 one, as it is defined now.

There seems to be an assumption that it is all or nothing, i.e. pass all AAA criteria or don't bother doing any.  That is how conformance works (for want of a more complex scoring approach), but it does not need to be how organisations apply the guidelines.

The advice I give clients asking about triple-A conformance is: "Let's do a review of them with the site in mind, and see which are most appropriate and feasible to apply."

The triple-A criteria tend to be harder to implement, harder to test, and may require updates to the design approach. That does not mean don't try, but it does mean working through how each would be applied in your context.

General regulations will generally require double-A conformance because it is known that not all triple-A criteria are applicable to every scenario. When you go beyond double-A, it should be a granular approach, not all-or-nothing.

Is there a place where higher-education institutions come together and set standards for themselves? If so, can you recommend there that they apply X, Y & Z triple-A criteria as a standard for them?

**Comment by GreggVan** [comment]
AAA are never required for exactly this reason.  Some of them cannot be applied to all content.  We leave them in so that developers will consider them  (as Alastair points out).  This one is both untestable in a way that has inter-rater reliability and also cannot be applied to all content.  But it is a very valuable one to have before people as they write.  Almost everything could be made simpler and clearer (though this is a pretty high bar).  In WCAG 3 I **_hope_** we can figure out a way to include these more 'recommendation - do as much as you can' items directly along with the 'testabel requirements that can be applied everywhere'   items so they get more visibility --- and where we can reward them with higher (Silver Gold) levels if they assert that they tried to do this as best they could.

**Comment by electronicwoft** [comment]
a  number of recommended tests such as  the Flesch-Kincaid ease of reading score might be useful for this success criterion to avoid cases in which  online tests of questionable validity are used either to contest or defend the readability of this or that content. Also, the measure of lower secondary education is both   ambiguous and much higher than many organisations such as the Australian government sets for itself in terms of readability. Perhaps as governments  defined contexts - not necessarily an  exhaustive list -  might make this criterion workable?

**Comment by jkshapiro** [comment]
I don't know that I'd call Flesch-Kincaid "recommended". More discussion at #4022 .
